device-registry/src/main/scala/org/genivi/sota/device_registry/Routes.scala

```diff
@@ -123,6 +123,10 @@ class Routes(namespaceExtractor: Directive1[Namespace],
         (get & path("group_info") & pathEnd) {
           listGroups(ns)
         } ~
+        (put & extractUuid & pathPrefix("group_info") & path("rename") & parameter('groupName.as[Name])) {
```

Comment:
No, path implies that it is the final segment. pathPrefix states that there are further segments to be matched, in this case "rename".